### PR TITLE
Add GitHub templates

### LINKED ISSUE #105: Add issue and PR templates
## Description
We need to add templates to ensure people writing/reviewing code are looking at the correct things

## Definition of done
- [ ] Templates are added to the codebase

### PR DESCRIPTION
**Pull request by heytherewill**
### Summary
This adds issue and PR templates

### Relationships
Closes #105 

## Merge permissions
Only myself

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+<!-- OPTIONAL: Are there other possible solutions that might seem more obvious? Tell us why you didn't go with those -->
+
+### Relationships
+<!-- MANDATORY: Mention any issues or PRs that are connected to this -->
+<!-- DO NOT OPEN A PULL REQUEST THAT DOES NO CLOSE/REF AN ISSUE -->
+Closes #
+
+## Repository health
+
+### Pull Request checklist
```
Comment (sofokli1) [inline .github/pull_request_template.md:21]: Is it possible to add an other check box for one of my OKRs. `- [ ] The code used custom templates`

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+## Info
```
Comment (sofokli1) [inline .github/pull_request_template.md:1]: what is the point of `info` when below is the `summary`?

Comment (heytherewill) [reply .github/pull_request_template.md:1]: Separating the while thing into sections. The `##` adds separators

File: .github/pull_request_template.md (hunk 1 above)
Comment (heytherewill) [reply .github/pull_request_template.md:21]: Sure, doable

**Pull request merged**